routes/index.js

```diff
@@ -4,6 +4,6 @@ import apiRouter  from './api';
 const router = express.Router();
 
 
-router.use("/api", apiRouter);
+router.use("/api/v1", apiRouter);
```

Comment:
you have changed the base route from "api" to "api/v1".
I think you should revert this so that everyone add their own version when creating a route ie : it should be /v1/login instead of /login

Reply:
This isn't accurate as it creates a lot of overhead for each person when defining a new route.
The base route should include the version number so we can ease the maintenance.
Follow the [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle.